setup.py

```diff
@@ -47,6 +47,12 @@
         "dev": [
             "pytest",
             "pytest-xdist"
+        ],
+        "unit_test": [
+            "pandas",
+            "pytest",
+            "pytest-xdist",
+            "python-dateutil"
```

Comment:
It's not for the CI, but for the ServerService_test.py. It was included by @MariusWirtz on #[b051c59](https://github.com/cubewise-code/tm1py/commit/b051c59711fd77b119bb15aef02a43e0e66d2a92)

Reply:
Understood. Then it would be a dev dependency as well. If we move it to the *dev* group, we would get rid of the additional *unit_test* group in the _dependency.yml_ What do you think about it?

Reply:
Agreed!

Reply:
Addressed in #1173